CMakeLists.txt

```diff
@@ -60,7 +60,9 @@ num_option(ONLY_PUBSUB_API "Only pubsub API" OFF)
 num_option(USE_PROXY "Use proxy" ON)
 num_option(USE_GZIP_COMPRESSION "Use gzip compression" ON)
 num_option(RECEIVE_GZIP_RESPONSE "Use gzip decompression" ON)
+num_option(USE_RETRY_CONFIGURATION "Use requests retry" ON)
 num_option(USE_SUBSCRIBE_V2 "Use subscribe v2" ON)
+num_option(USE_SUBSCRIBE_EVENT_ENGINE "Use Subscribe Event Engine" ON)
```

Comment:
Does this flag influent current users of the SDK?

Reply:
This is temporary while was developing or IDE won't highlight the code properly :/ 
By default, Subscribe EE and retry will be off.